### DISCUSSION TITLE
Downloading en module in pacy v3.5.2

### DISCUSSION THREAD

**Discussion by akesh1235** [explosion/spaCy#12603] [Help: Installation] [answered]
When en module is being downloaded facing an error that is **[!] As of spaCy v3.0, shortcuts like 'en' are deprecated. Please use the full
pipeline package name 'en_core_web_sm' instead.** i want to use spaCy  with errant.load('en')

**Comment by adrianeboyd** [comment]
I think that ERRANT only supports spacy v2, so try installing `errant` again, which should make sure you have the right version of spacy.

**Reply by akesh1235** [reply]
Yeah, I tried installing ERRANT with Spacy v2 still there's the same problem.

**Reply by adrianeboyd** [reply]
`en` should still work as a model name in v2, so maybe there is still a problem with your spacy install? Without more information from a traceback and maybe also info from `conda list` or `pip list` it's hard to know for sure what's going on.

To double-check, you can try installing the most recent version of spacy v2 with `pip install 'spacy~=2.0'`, which should currently install v2.3.9.

**Reply by akesh1235** [reply, marked as answer]
Thank you for your help sir, 
**We can use errant with Spacy v3** by updating errant 
https://github.com/chrisjbryant/errant/pull/41/commits/e53a16a083a64a95cffffde56828c4d4ddab62ad
